src/Symfony/Component/Notifier/Bridge/Lox24/Tests/Lox24TransportTest.php

```diff
@@ -304,18 +304,24 @@ private function assertRequestBody(
         array $bodyOverride = [],
         array $headersOverride = [],
         int $responseStatus = 200,
-        array $responseContent = [],
+        array $responseContent = []
     ): void {
-        $body = array_merge(self::REQUEST_BODY, $bodyOverride);
+        $expectedBody = json_encode(array_merge(self::REQUEST_BODY, $bodyOverride));
```

Comment:
I would love to see MockHttpClient using MockResponse here for the tests 🥰

Reply:
@OskarStark done. please check